sdk/scheduler/src/test/java/com/mesosphere/sdk/config/validate/RegionCannotChangeTest.java

```diff
@@ -0,0 +1,113 @@
+package com.mesosphere.sdk.config.validate;
+
+import com.mesosphere.sdk.specification.DefaultServiceSpec;
+import com.mesosphere.sdk.specification.PodSpec;
+import com.mesosphere.sdk.specification.ServiceSpec;
+import com.mesosphere.sdk.testutils.TestConstants;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+import static org.mockito.Mockito.when;
+
+public class RegionCannotChangeTest {
```

Comment:
I think we're missing the "had a region, but it was removed" test.

Reply:
Yep, will add.